lib/components/app/locale-selector.tsx

```diff
@@ -44,31 +44,42 @@ const LocaleSelector = (props: LocaleSelectorProps): JSX.Element => {
   }
 
   return (
-    <NavDropdown
+    <Dropdown
       id="locale-selector"
-      pullRight
+      // TODO: How to make this work without block ruby?
+      style={{ display: 'block ruby' }}
       title={
-        <StyledIconWrapper style={{ color: 'rgba(255, 255, 255, 0.85)' }}>
+        <span
+          style={{
+            color: 'rgba(255, 255, 255, 0.85)'
+          }}
+        >
           <GlobeAmericas />
-        </StyledIconWrapper>
+        </span>
       }
     >
-      {Object.keys(configLanguages)
-        .filter((locale) => locale !== 'allLanguages')
-        .map((locale) => (
-          <MenuItem
-            className="locale-name"
-            key={locale}
-            onClick={(e: MouseEvent) => handleLocaleSelection(e, locale)}
-          >
-            <span
-              style={locale === currentLocale ? { fontWeight: 'bold' } : {}}
+      <ul style={{ listStyle: 'none', margin: 0, padding: 0 }}>
+        {Object.keys(configLanguages)
+          .filter((locale) => locale !== 'allLanguages')
+          .map((locale) => (
+            <li
+              aria-selected={locale === currentLocale}
+              key={locale}
+              onClick={(e: MouseEvent) => handleLocaleSelection(e, locale)}
+              onKeyPress={(e: any) => handleLocaleSelection(e, locale)}
+              // We are correct, not eslint: https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html
+              // eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role
+              role="option"
```

Comment:
The dropdown widget is in essence a button that controls/reveals the dropdown content, so it is not a combo box, and thus this role is not needed. Each item under this list should be a button instead.

Reply:
The link I referred to disagrees. This might be an item to discuss with everyone else.

Reply:
Yeah but this is not a combo box.

Reply:
Have removed the combobox. I think I've got things working well....